project/Dependencies.scala

```diff
@@ -23,7 +23,7 @@ object Dependencies {
   val agronaVersion = "1.15.1"
   val nettyVersion = "3.10.6.Final"
   val protobufJavaVersion = "3.16.1"
-  val logbackVersion = "1.2.11"
+  val logbackVersion = "1.2.19"
```

Comment:
Seems 1.2.11 is already the latest version (https://repo1.maven.org/maven2/ch/qos/logback/logback-classic/)

Reply:
(seems like we've been on 1.2.9+ since Akka 2.6.18)

Reply:
Ah, indeed, not sure why I thought it would be .19.